libr/anal/fcn.c

```diff
@@ -2472,9 +2478,10 @@ static bool can_affect_bp(RAnal *anal, RAnalOp* op) {
 R_API void r_anal_fcn_check_bp_use(RAnal *anal, RAnalFunction *fcn) {
 	RListIter *iter;
 	RAnalBlock *bb;
+#if 0
```

Comment:
just delete those lines if you don't want them, no need for `#if 0`

Reply:
I have left that because code like "push rax, pop rbp" should be handled there as well, but it's difficult to detect all such cases, so it could be implemented in the future, but you're probably right - there's no need for it now. ;)